service/src/main/resources/db/changesets/20230530_avoid_postgres_specific_features.yaml

```diff
@@ -19,12 +20,12 @@ databaseChangeLog:
                   remarks: Deleting a study will cascade to delete its properties
               - column:
                   name: key
```

Comment:
Is it possible to change the name of this column from `key` to `property_key`? Obviously need java changes for the DAOs.